src/components/Work/Tabs/MoreInformation.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { getCollection } from "api/elasticsearch-api";
+
+function WorkTabsMoreInformation({ collection = {} }) {
+  if (Object.keys(collection).length === 0) return null;
+  const [adminEmail, setAdminEmail] = React.useState();
+
+  React.useEffect(() => {
+    async function fn() {
+      let collectionResponse = await getCollection(collection.id);
+      setAdminEmail(collectionResponse._source.adminEmail || "");
+    }
+    fn();
+  }, []);
+
+  return adminEmail ? (
+    <>
+      <h4>More Information</h4>
+      <p>
+        For more information on this item or collection, please contact{" "}
+        <a href={`mailto:${adminEmail}`}>{adminEmail}</a>
```

Comment:
Do we obfuscate email addresses these days?

Reply:
Yea, good idea.